src/pages/account/[address].tsx

```diff
@@ -0,0 +1,109 @@
+import { Container, Divider, VStack, Box, Link } from '@chakra-ui/react';
+import { Trans, t } from '@lingui/macro';
+import { graphql, Link as GatsbyLink, withPrefix } from 'gatsby';
+import type { FunctionComponent } from 'react';
+import { getAddress } from 'viem';
+
+import banner from '../../assets/images/seo/home.png';
+import {
+  AccountProfileBanner,
+  AccountProfileTabs,
+  AccountInfo,
+} from '../../features/account';
+import type { Fields } from '../../utils';
+import NotFound from '../404';
+
+type AccountPageProps = {
+  params: {
+    address: string;
+  };
+};
+
+const AccountPage: FunctionComponent<AccountPageProps> = ({ params }) => {
+  let { address } = params;
+  try {
+    address = getAddress(params.address);
+  } catch {
+    return (
+      <Box data-testid="account-profile-not-found">
+        <NotFound />
+      </Box>
+    );
+  }
+
+  return (
+    <Box position="relative" data-testid="account-profile">
+      <AccountProfileBanner />
+      <Container maxWidth="container.xl" paddingTop="0" position="relative">
+        <VStack mt="175" spacing={['10', null, '20']}>
+          <AccountInfo address={address} />
+          <Box position={['static', null, 'absolute']} right="5" top="100">
```

Comment:
Why is the position different depending on screen size?

Reply:
edit button floating on right side of the screen when it is large screen
edit button should become static center after account infor when it is small screen